cli/cmd/create_project.go

```diff
@@ -25,7 +26,7 @@ type createProjectCmdParams struct {
 var createProjectParams *createProjectCmdParams
 
 const gitErrMsg = `Please specify a 'git-user', 'git-token', and 'git-remote-url' as flags for configuring a Git upstream repository`
-const gitMissingUpstream = `WARNING: Creating a project without Git upstream repository is not recommended. 
+const gitMissingUpstream = `WARNING: Creating a project without Git upstream repository is not recommended and will not be supported in the future anymore.
```

Comment:
I think we should be a bit more specific here, e.g., stating which version we plan to deprecate this support.
Though you have implemented it according to #6306, so I'll approve this.

Reply:
I can add a specific version there, it is not a problem, but do we already know in which version we want to deprecate the support ?